packages/tools/generator-pie-component/src/app/templates/src/defs-react.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+/**
+ * TODO: Verify if ReactBaseType can be set as a more specific React interface
```

Comment:
Please can we create a ticket to investigate/do this?

Reply:
In fact, that is a reminder, not a leftover. With that, we can easily identify if the React base type wasn't taken care of for any generated  component.